driver/normalizer/normalizer.go

```diff
@@ -72,81 +72,110 @@ func identifierWithPos(nameVar string) ObjectOp {
 	})
 }
 
-func tokenIsIdentifier(typ, tokenKey string, roles ...role.Role) Mapping {
-	return AnnotateType(typ, MapObj(
-		Fields{
-			{Name: tokenKey, Op: Var("name")},
-		},
-		Fields{
-			{Name: tokenKey,
-			 Op: UASTType(uast.Identifier{}, Obj{
-			 	"Name": Var("name"),
-			 })},
+func mapStr(nativeType string) Mapping {
```

Comment:
This name is telegraphic enough that I think this function deserves a brief comment.

Reply:
Sorry, Go convention: `// mapStr ...`

Reply:
It's okay to relax that for unexported names (though it certainly helps if they later get exported).